tests/trello/gui/resources/BoardsPage.resource

```diff
@@ -7,9 +7,29 @@ Resource         TrelloGUI.resource
 ${BOARDS_PATH}          boards
 ### Locators ###
 ${YOUR_WORKSPACES}             css:h3.boards-page-section-header-name
+${CREATE_NEW_BOARD_BUTTON}             xpath://div[@class="boards-page-board-section mod-no-sidebar"]//div[@class="board-tile mod-add"]
+${BOARD_NAME_INPUT}             xpath://div[@title='Create board']/ancestor::section//input[@type="text"]
+${CREATE_BUTTON}             xpath://div[@title='Create board']/ancestor::section//button[text()='Create']
 
 *** Keywords ***
 Verify Current Page Is Boards Page
     Wait Until Location Contains     ${BOARDS_PATH}
     Wait Until Element Is Visible    ${YOUR_WORKSPACES}
     Element Text Should Be           ${YOUR_WORKSPACES}    YOUR WORKSPACES
+
+Click New Board Button
+    Wait Until Location Contains     ${BOARDS_PATH}
+    Wait Until Element Is Visible    ${CREATE_NEW_BOARD_BUTTON} 
+    Click Element           ${CREATE_NEW_BOARD_BUTTON}
+      
+
+Set Name to Board
+    [Documentation]    Creates a new Board
```

Comment:
Documentation says "Creates a new Board" but the name is "Set Name to Board", and it can't be used by itself to create a board... but it also confirms the creation after setting the name, right?
Maybe you can have a keyword "Create A Board" for the whole workflow, or separate keywords, but taking into account that the names, responsibilities, and documentation should be accurate.